qutip/solver/multitraj.py

```diff
@@ -193,13 +205,16 @@ def run(self, state, tlist, ntraj=1, *,
         result.stats['run time'] = time() - start_time
         return result
 
-    def _run_one_traj(self, seed, state, tlist, e_ops):
+    def _run_one_traj(self, seed, state, tlist, e_ops,
+                      no_jump=False, jump_prob_floor=0.0):
```

Comment:
`MultiTrajSolver` also apply to stochastic evolution which have continuous noise, no jump...
These new input do not belong here, but in `MCSolverImprovedSampling`.